tests/data_context/test_data_context_datasources.py

```diff
@@ -96,125 +90,6 @@ def test_data_context_instantiates_inline_store_backend_with_filesystem_config(
-    of a cache miss.
-    """
-    context = in_memory_runtime_context
-
-    name = "my_fake_datasource_name"
-
-    # Initial GET will miss the cache, necessitating store retrieval
-    with mock.patch(
-        "great_expectations.datasource.datasource_dict.DatasourceDict.__getitem__"
-    ) as mock_get:
-        context.get_datasource(name)
-
-    assert mock_get.called
-
-    # Subsequent GET will retrieve from the cache
-    with mock.patch("great_expectations.data_context.store.DatasourceStore.get") as mock_get:
-        context.get_datasource(name)
-
-    assert not mock_get.called
-
-
-@pytest.mark.unit
-def test_BaseDataContext_add_datasource_updates_cache(
-    in_memory_runtime_context: EphemeralDataContext,
-    pandas_enabled_datasource_config: dict,
-) -> None:
-    """
-    What does this test and why?
-
-    For persistence-disabled contexts, we should only update the cache upon adding a
-    datasource.
-    """
-    context = in_memory_runtime_context
-
-    name = pandas_enabled_datasource_config["name"]
-
-    assert name not in context.datasources
-
-    context.add_datasource(**pandas_enabled_datasource_config)
-
-    assert name in context.datasources
-
-
-@pytest.mark.unit
-def test_BaseDataContext_update_datasource_updates_existing_data_source(
-    in_memory_runtime_context: EphemeralDataContext,
-    pandas_enabled_datasource_config: dict,
-) -> None:
-    """
-    What does this test and why?
-
-    Updating a Data Source should update a Data Source
-    """
-    context = in_memory_runtime_context
-
-    name = context.list_datasources()[0]["name"]
-    pandas_enabled_datasource_config["name"] = name
-    data_connectors = pandas_enabled_datasource_config["data_connectors"]
-    pandas_enabled_datasource_config.pop("class_name")
-    datasource = Datasource(**pandas_enabled_datasource_config)
-
-    assert name in context.datasources
-    cached_datasource = context.datasources[name]
-    assert cached_datasource.data_connectors.keys() != data_connectors.keys()  # type: ignore[union-attr]
-
-    context.update_datasource(datasource)
-
-    retrieved_datasource = context.get_datasource(datasource_name=name)
-    assert retrieved_datasource.data_connectors.keys() == data_connectors.keys()  # type: ignore[union-attr]
-
-
-@pytest.mark.unit
-def test_BaseDataContext_update_datasource_fails_when_datsource_does_not_exist(
-    in_memory_runtime_context: EphemeralDataContext,
-    pandas_enabled_datasource_config: dict,
-) -> None:
-    """
-    What does this test and why?
-
-    Updating a data source that does not exist should create a new data source.
-    """
-    context = in_memory_runtime_context
-
-    name = pandas_enabled_datasource_config["name"]
-    pandas_enabled_datasource_config.pop("class_name")
-    datasource = Datasource(**pandas_enabled_datasource_config)
-
-    assert name not in context.datasources
-
-    with pytest.raises(DatasourceNotFoundError):
-        context.update_datasource(datasource)
-
-
```

Comment:
just confirming that these are being removed because we don't support these methods on the data context.

Reply:
Yup, I think there were 2 files that tested all this crud on context, but since we'll remove these methods in https://greatexpectations.atlassian.net/browse/V1-321, I don't think there's sense in taking the time to port them over to FDS